gym_roboy/envs/msj_env.py

```diff
@@ -95,3 +97,10 @@ def _did_reach_goal(self, current_state: MsjRobotState) -> bool:
     @property
     def _l2_distance_for_success(self):
```

Comment:
in the function "_did_reach_goal" the return boolean should check if the current joint velocity is "sufficient" close to the target joint velocity.

Reply:
can we write instead of `return bool(l2_distance < self._l2_distance_for_success)`
`return l2_distance < self._l2_distance_for_success` ?

Reply:
agree, the velocity must also be checked.

Reply:
if we dont wrap the calculation in a bool(...) then it returns a numpy.bool, which returns False on isinstance(x, bool)